### PR TITLE
Document Backups

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/docs/products/databases/backups/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+layout: article
+title: Database Backups
```
Comment (eldadfux) [inline src/routes/docs/products/databases/backups/+page.markdoc:3]:
```suggestion
title: Backups
```

I think we can go with the simpler title here.

File: src/routes/docs/products/databases/backups/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Learn how to efficiently back up your databases, ensuring data security and smooth recovery.
+
+{% info title="Backups are an Appwrite Cloud feature and available on the Enterprise plan" %}
+{% /info %}
+
+Appwrite Backups enable you to automate database backups using Backup Policies, with support for both pre-defined and custom retention options. You can also create manual backups whenever necessary.
+
+# Backups {% #backups %}
+
+## Create Backups {% #create-backups %}
```
Comment (eldadfux) [inline src/routes/docs/products/databases/backups/+page.markdoc:17]: All of our titles should be sentence case per our writing style guide.

File: src/routes/docs/products/databases/backups/+page.markdoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Appwrite Backups enables seamless, **encrypted** database backups on Cloud.
+Learn how to efficiently back up your databases, ensuring data security and smooth recovery.
+
+{% info title="Backups are an Appwrite Cloud feature and available on the Enterprise plan" %}
+{% /info %}
+
+Appwrite Backups enable you to automate database backups using Backup Policies, with support for both pre-defined and custom retention options. You can also create manual backups whenever necessary.
+
+# Backups {% #backups %}
```
Comment (eldadfux) [inline src/routes/docs/products/databases/backups/+page.markdoc:15]: This title is repetitive we can skip it.

File: src/routes/docs/products/databases/backups/+page.markdoc [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+description: Learn how to efficiently back up your databases on Appwrite Cloud, ensuring data security and seamless recovery.
+---
+
+Appwrite Backups enables seamless, **encrypted** database backups on Cloud.
+Learn how to efficiently back up your databases, ensuring data security and smooth recovery.
+
+{% info title="Backups are an Appwrite Cloud feature and available on the Enterprise plan" %}
+{% /info %}
+
+Appwrite Backups enable you to automate database backups using Backup Policies, with support for both pre-defined and custom retention options. You can also create manual backups whenever necessary.
```
Comment (eldadfux) [inline src/routes/docs/products/databases/backups/+page.markdoc:13]: Not just custom retention options. Retention is only one setting in a policy.

File: src/routes/docs/products/databases/backups/+page.markdoc (hunk 2 above)
Comment (eldadfux) [inline src/routes/docs/products/databases/backups/+page.markdoc:17]: This is a section about policies. I would adjust the title, start by explaining what a policy is, and then have another sub-heading with the steps to create a policy. 

The same way we might want to add a section for manual backups.

File: src/routes/docs/products/databases/backups/+page.markdoc [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+## Create Backups {% #create-backups %}
+
+To automate your database backups, you first need to create Backup Policies that run at scheduled intervals.
+
+1. In the Appwrite Console's sidebar, click **Databases**
+2. Select & navigate to your database and click on the **Backups** Tab
+3. Click on **Create Policies** & adjust your settings or select a pre-defined policy
+4. Click on **Create**
+
+Your database is now set up for automated backups with just a few clicks.\
```
Comment (eldadfux) [inline src/routes/docs/products/databases/backups/+page.markdoc:26]:
```suggestion
Your database is now set up for automated backups with just a few clicks.
```

File: src/routes/docs/products/databases/backups/+page.markdoc [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+To ensure your backups are robust and effective, consider the following best practices:
+
+1. **Schedule Regular Backups**:
+   Add multiple backup policies based on the frequency of database changes. Daily or weekly backups are often sufficient for most use cases.
+
+2. **Retain Critical Backups Longer**:
+   Use custom retention on Policies to keep backups of critical data for extended periods, ensuring historical records are available when needed.
+
+3. **Monitor Backup Status**:
+   Regularly check the backup status in the Backups Tab to ensure no scheduled backups have failed.
```
Comment (eldadfux) [inline src/routes/docs/products/databases/backups/+page.markdoc:53]: Let's make sure we log errors when a backup fails, this shouldn't really be an option for users. This should be our responsibility to check not the developer's.

File: src/routes/docs/products/databases/backups/+page.markdoc [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+   All backups are stored in an **encrypted** format to protect your data.
+
+4. **Optimize Backup Policies Based on Data Sensitivity**:
+   Tailor your backup frequency and retention settings according to the sensitivity and importance of the data.
+   Critical data may require more frequent backups, while less essential data can have longer retention and fewer backups.
```
Comment (eldadfux) [inline src/routes/docs/products/databases/backups/+page.markdoc:58]: I miss a section where we can provide some important reassurances like the fact all backups are stored in a remote location, and all backups are encrypted.

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: src/routes/docs/products/databases/backups/+page.markdoc [hunk 8]
```diff
@@ -0,0 +1,58 @@
+---
+layout: article
+title: Database Backups
+description: Learn how to efficiently back up your databases on Appwrite Cloud, ensuring data security and seamless recovery.
+---
+
+Appwrite Backups enables seamless, **encrypted** database backups on Cloud.
+Learn how to efficiently back up your databases, ensuring data security and smooth recovery.
+
+{% info title="Backups are an Appwrite Cloud feature and available on the Enterprise plan" %}
```
Comment (eldadfux) [inline src/routes/docs/products/databases/backups/+page.markdoc:10]:
```suggestion
{% info title="Backups are available on Appwrite Cloud for all Pro, Scale, and Enterprise customers." %}
```

**Review by Herry0092 — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**